centreon/www/front_src/src/Dashboards/components/DashboardPlaylists/PlaylistsListing/Columns/Name.tsx

```diff
@@ -0,0 +1,26 @@
+import { isNil } from 'ramda';
+import { useNavigate } from 'react-router';
+
+import { Box } from '@mui/material';
+
+import { ComponentColumnProps } from '@centreon/ui';
+
+const Name = ({ row }: ComponentColumnProps): JSX.Element => {
+  const navigate = useNavigate();
+
+  const { name, role, id } = row;
+
+  const isNestedRow = !isNil(role);
+
+  if (isNestedRow) {
+    return <Box />;
+  }
+
+  const linkToPlaylist = (): void => {
+    navigate(`/home/dashboards/playlists/${id}`);
```

Comment:
```suggestion
    navigate(generatePath(routeMap.dashboard, { layout: DashboardLayout.Playlist, dashboardId: id }));
```
generatePath comes from react-router